install/prepare_site_init.md

```diff
@@ -133,51 +131,40 @@ with system-specific customizations.
 
 1.  To federate Keycloak with an upstream LDAP:
 
-    In the example below, the LDAP server has the hostname `dcldap2.us.cray.com` and is using the port 636.
+    1. Set environment variables for LDAP and PORT
 
-    ```bash
-    linux# export LDAP=dcldap2.us.cray.com
-    linux# export PORT=636
-    ```
+       In the example below, the LDAP server has the hostname `dcldap2.us.cray.com` and is using the port 636.
+
+       ```bash
+       linux# export LDAP=dcldap2.us.cray.com
```

Comment:
Can you pretty please remove the `linux#` prompts? If this needs to be ran on an _external_ machine, use `external#` but with the new `main` branch convention: https://github.com/Cray-HPE/docs-csm/blob/MTL-1695/introduction/documentation_conventions.md#step-examples